Build fails due to broken dependency for nats-io/nkeys

`
$ make
go build -ldflags "-X=main.Build=086a622 -X=main.BuildTime=`date -u '+%Y-%m-%d_%I:%M:%S%p'` -X=main.Version=v1.1.0" -o versitygw cmd/versitygw/*.go
go: downloading github.com/nats-io/nkeys v0.4.14
/home/x/go/pkg/mod/github.com/nats-io/nats.go@v1.48.0/nats.go:43:2: reading github.com/nats-io/nkeys/go.mod at revision v0.4.14: unknown revision v0.4.14
make: *** [Makefile:39: versitygw] Error 1
`

Dependency was updated with this commit: https://github.com/versity/versitygw/commit/ca2dd9b4b3b7cfd6c5f1504618b6aef27cfda669
However v0.4.14 does not exist in https://github.com/nats-io/nkeys/tags

--- a/go.mod
+++ b/go.mod
@@ -55,7 +55,7 @@ require (
 	github.com/klauspost/cpuid/v2 v2.3.0 // indirect
 	github.com/kylelemons/godebug v1.1.0 // indirect
 	github.com/mitchellh/go-homedir v1.1.0 // indirect
-	github.com/nats-io/nkeys v0.4.14 // indirect
+	github.com/nats-io/nkeys v0.4.12 // indirect
 	github.com/nats-io/nuid v1.0.1 // indirect
 	github.com/pierrec/lz4/v4 v4.1.25 // indirect
 	github.com/pkg/browser v0.0.0-20240102092130-5ac0b6a4141c // indirect

